go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.6.2
 	golang.org/x/crypto v0.0.0-20200302210943-78000ba7a073
+	golang.org/x/tools v0.0.0-20200329025819-fd4102a86c65
```

Comment:
why that ?

Reply:
ok, I see. It's for goimports used during build. Does go has the notion of a "developer dependency" (like npm or Maven) ? I'm not big fan to mix in dependencies here which are not part of the application itself.